### ISSUE TITLE
AoC: Show personal position for daystar command

### ISSUE THREAD

**Issue by ScientificGuitar** [python-discord/sir-robin#72]
Currently the daystar command only shows the fastest completions for a day and star and not where your personal stats are compared to them. It would be nice to see your personal stats, similar to the way it shows in the leaderboard command.

**Comment by ChrisLovering** [comment]
Agreed, this seem like a useful enhancement. Were you interested in looking to implement this yourself, or shall I mark it as up for grabs for other contributors?
